week-3/Homework/mandatory/1-oxygen-levels.js

```diff
@@ -9,32 +9,36 @@ To be safe, they need to land on the first unamed planet that has Oxygen levels
 Write a function that finds the oxygen level of the first safe planet - Oxygen between 19.5% and 23.5%
 */
 
-function safeLevels() {
+function checkLevels(planet) {
+  return parseFloat(planet) > 19.5 && parseFloat(planet) < 23.5;
```

Comment:
This parses `planet` from a string to a float twice - can you make it so that we only parse the number once?